### PR TITLE
Rss Reader

### PR DESCRIPTION
**Pull request by RonjaPonja**

### CODE REVIEW

File: apps/rss/rss.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        self.daemon = True
+
+    def run(self):
+        while True:
+            log.debug("Updating Feeds")
+            update()
+            time.sleep(3*60)
+
+
+with edi.Manager(name="RSS", descr="RSS feed client.") as e:
```
Comment (irq0) [inline apps/rss/rss.py:150]: The command names suggest something more than a rss reader. I suggest something like rss-sub, rss-unsub, rss-list

File: apps/rss/rss.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                 descr="Un-Track RSS feed")
+    def del_rss_recv(**args):
+        if len(args["args"].split()) == 1:
+            del_rss(args["src"], args["user"], args["args"].split()[0])
+        else:
+            to_send = "Try it like this: !unsub <alias>"
+            publish(args["src"], args["user"], to_send)
+
+    @edi.edi_cmd(e, "list",
+                 args="TEXT",
```
Comment (irq0) [inline apps/rss/rss.py]: does list really have args?. If none use "NONE"

File: apps/rss/rss.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        while True:
+            log.debug("Updating Feeds")
+            update()
+            time.sleep(3*60)
+
+
+with edi.Manager(name="RSS", descr="RSS feed client.") as e:
+
+    @edi.edi_cmd(e, "sub",
+                 args="TEXT",
```
Comment (irq0) [inline apps/rss/rss.py:153]: Not yet used anywhere else, but I think using args=URL would be beneficial later

**Comment by RonjaPonja** [comment]
I did as I was told.

File: apps/rss/rss.py (hunk 3 above)
Comment (RonjaPonja) [inline apps/rss/rss.py:153]: It's a URL and a Alias though. Can this reasonably be "ALIAS URL"?

**Pull request merged**